rule-packs/azure.json

```diff
@@ -82,7 +82,7 @@
         "queries": [
           {
             "name": "query0",
-            "query": "find azure_policy_assignment with id $=\"/SecurityCenterBuiltIn\" and params.vulnerabilityAssesmentMonitoringEffect=\"Disabled\"",
+            "query": "find azure_policy_assignment with id $=\"/SecurityCenterBuiltIn\" and parameters.vulnerabilityAssesmentMonitoringEffect=\"Disabled\"",
```

Comment:
you can also change `vulnerabilityAssesmentMonitoringEffect` -> `vulnerabilityAssessmentMonitoringEffect`